app.py

```diff
@@ -67,33 +71,42 @@ def register():
             flash("Username already exists")
             return render_template("register.html")
 
+@app.route("/profile", methods=["GET"])
+def profile():
+    return render_template("profile.html")

```

Comment:
profile requires display of a user's blogs
make a backend method to take in a user as a paramter and return something similar to blogs.describe
@yevagorbachev